biogtr/data_structures.py

```diff
@@ -56,46 +56,58 @@ def __init__(
         else:
             self._skeleton = skeleton
 
-        if not isinstance(bbox, torch.Tensor):
+        if bbox is None:
+            self._bbox = torch.empty(1, 0, 4)
+
+        elif not isinstance(bbox, torch.Tensor):
             self._bbox = torch.tensor(bbox)
+
         else:
             self._bbox = bbox
 
+        if self._bbox.shape[0] and len(self._bbox.shape) == 1:
```

Comment:
Nit: prefer explicit over implicit boolean conversions, especially with numeric types, i.e.:

```suggestion
        if self._bbox.shape[0] == 0 and len(self._bbox.shape) == 1:
```

Reply:
will start doing this from now on!